tests/phpunit/tests/block-supports/typography.php

```diff
@@ -429,8 +434,7 @@ public function data_generate_font_size_preset_fixtures() {
 			),
 			'returns already clamped value'              => array(
 				'font_size_preset' => array(
-					'size'  => 'clamp(21px, 1.313rem + ((1vw - 7.68px) * 2.524), 42px)',
-					'fluid' => false,
+					'size' => 'clamp(21px, 1.313rem + ((1vw - 7.68px) * 2.524), 42px)',
```

Comment:
Addressing an omission - here the tests is checking whether the clamped value is returned when fluid typography is enabled.